pkg/installmanager/installmanager.go

```diff
@@ -78,6 +78,8 @@ const (
 	defaultPullSecretMountPath          = "/pullsecret/" + corev1.DockerConfigJsonKey
 	defaultManifestsMountPath           = "/manifests"
 	defaultHomeDir                      = "/home/hive" // Used if no HOME env var set.
+	sshPrivKeyPathEnvVar                = "SSH_PRIV_KEY_PATH"
```

Comment:
Should we put these definitions in a common place that can be used by `pkg/install/generate.go` as well.